.github/workflows/scripts/releaseNotesGenerator.sh

```diff
@@ -0,0 +1,19 @@
+JIRA_TOKEN=$1
+JIRA_FIXED_VERSION=$2
+
+fixed_version_found=false
+curl -X GET -H "Authorization: Basic $JIRA_TOKEN=" https://appsflyer.atlassian.net/rest/api/3/project/11723/versions | jq -r '.[] | .name+""+.id' | while read version ; do
+if [[ "$version" == *"$JIRA_FIXED_VERSION"* ]] ;then
+    echo "$JIRA_FIXED_VERSION Found!"
+    fixed_version_found=true
+    version_id=${version#"$JIRA_FIXED_VERSION"}
+    echo $(curl -X GET -H "Authorization: Basic $JIRA_TOKEN=" https://appsflyer.atlassian.net/rest/api/3/search?jql=fixVersion=$version_id | jq -r '.issues[] | "- " + .fields["summary"]+"@"') > "$JIRA_FIXED_VERSION-releasenotes".txt
```

Comment:
Does it work for you? I know that jira deprecated basic auth for their api (i changed it in [react native](https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/blob/master/.github/workflows/scripts/releaseNotesGenerator.sh#L5)). see more 
[here](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/) and [here](https://community.atlassian.com/t5/Jira-questions/Basic-authentication-with-passwords-is-deprecated/qaq-p/1427287)